### PR TITLE
Updated hammer_cli_foreman_webhooks to 0.0.4

### PR DESCRIPTION
**Pull request by ofedoren**
Supported Versions:

 * Nightly
 * 3.3

### CODE REVIEW

**Comment by ehelms** [comment]
[test deb]

**Review by ekohl — CHANGES_REQUESTED** [review]
There's a merge conflict now.

**Comment by ekohl** [comment]
[test deb]

**Comment by ehelms** [comment]
[test deb]

File: dependencies/focal/hammer_cli_foreman_webhooks/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ruby-hammer-cli-foreman-webhooks (0.0.3-1) stable; urgency=low
```
Comment (evgeni) [inline dependencies/focal/hammer_cli_foreman_webhooks/changelog:1]:
```suggestion
ruby-hammer-cli-foreman-webhooks (0.0.4-1) stable; urgency=low

  * 0.0.4 released

 -- Evgeni Golov <evgeni@debian.org>  Tue, 18 Oct 2022 10:44:53 +0200

ruby-hammer-cli-foreman-webhooks (0.0.3-1) stable; urgency=low
```

**Comment by evgeni** [comment]
needs https://github.com/theforeman/foreman-packaging/pull/8596

**Comment by evgeni** [comment]
which is now built, so
[test deb]

**Pull request merged**